app/templates/direct-award/save-search.html

```diff
@@ -21,32 +21,48 @@
 
 {% block main_content %}
 
-<div class="index-page">
-  <header class="page-heading">
-    <h1>
-      Save your search
-    </h1>
-  </header>
-  <p>{{ search_summary.markup() }}</p>
+<div class="single-question-page save-search-page">
   <div class="grid-row">
-    <form method="post" action="{{ url_for('direct_award.project_create', framework_framework=framework_framework) }}">
-      <input type="hidden" name="csrf_token" value="{{ csrf_token() }}" />
-      <input type="hidden" name="search_api_url" value="{{ search_api_url }}"/>
-      {%
-        with
-        question = "Name a new project",
-        name = "project_name"
-      %}
-        {% include "toolkit/forms/textbox.html" %}
-      {% endwith %}
-      {%
-        with
-        type = "save",
-        label = "Create project and save search"
-      %}
-        {% include "toolkit/button.html" %}
-      {% endwith %}
-    </form>
+    <div class="column-two-thirds">
+      <header class="page-heading-smaller page-heading-without-breadcrumb">
+        <h1>
+          Save your search
+        </h1>
+      </header>
+    </div>
   </div>
+  <form method="post" action="{{ url_for('direct_award.project_create', framework_framework=framework_framework)}}" id="createProjectForm">
+    <div class="grid-row">
+      <div class="column-two-thirds">
+        <div class="search-results-summary">
+          <p>{{ search_summary.markup() }}</p>
```

Comment:
http://alphagov.github.io/digitalmarketplace-frontend-toolkit/search-summary.html

we should use this method

Reply:
I'm not sure we should - it's one of the tiniest most pointless templates I've ever seen :)